boards/common/nrf52xxxdk/dist/openocd.cfg

```diff
@@ -0,0 +1,16 @@
+transport select swd
+
+source [find target/nrf52.cfg]
+
+# Work-area is a space in RAM used for flash programming
+# By default use 16kB
+if { [info exists WORKAREASIZE] } {
+   set _WORKAREASIZE $WORKAREASIZE
+} else {
+   set _WORKAREASIZE 0x4000
+}
+
+$_TARGETNAME configure -work-area-phys 0x20000000 -work-area-size $_WORKAREASIZE -work-area-backup 0
+
+flash bank $_CHIPNAME.flash nrf5 0x00000000 0 1 1 $_TARGETNAME
```

Comment:
Just "nrf5" without a number?

Reply:
Yes it's like this on my openocd version. Let's try with an explicit target name, I just pushed this change and it still works for me.